web/app/common/fetcher.ts

```diff
@@ -39,7 +48,9 @@ const fetcher = methods.reduce<Partial<FetcherObject>>((acc, method) => {
     };
 
     if (body) {
-      if (typeof body === 'object') {
+      if (contentType === 'multipart/form-data' && body instanceof FormData) {
```

Comment:
Maybe  `contentType === 'multipart/form-data'` will be enough. Look like  `body instanceof FormData` is redundant.

Reply:
you're right